runtime/src/main/java/org/corfudb/runtime/clients/SequencerHandler.java

```diff
@@ -37,6 +40,16 @@ public SequencerClient getClient(long epoch) {
     public ClientMsgHandler msgHandler = new ClientMsgHandler(this)
             .generateHandlers(MethodHandles.lookup(), this);
 
+    @ClientHandler(type = CorfuMsgType.SEQUENCER_METRICS_RESPONSE)
+    private static Object handleMetricsResponse(CorfuPayloadMsg<byte[]> msg,
+                                                ChannelHandlerContext ctx, IClientRouter r) {
+        try {
+            return SequencerMetrics.parseFrom(msg.getPayload());
+        } catch (InvalidProtocolBufferException e) {
+            throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)